packages/roosterjs-editor-dom/lib/table/tableMetadata.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Selectors for Table Selection
+ */
+export const enum TableMetadata {
```

Comment:
For now let's do not add this enum. I think in this PR only TABLE_CELL_SELECTED and TABLE_SELECTED is used, right?

We can defined them in where they are used and do not export. Later we can move them. 

Adding new type/interface/enum is easy, removing/changing is hard.